am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/helpers/DatabaseHelperRepository.java

```diff
@@ -30,6 +30,10 @@
     @RegisterConstructorMapper(Role.class)
     Role getRole(String roleName);
 
+    @SqlQuery("select attribute from access_management "
+        + "where access_management.resource_id = :resourceId")
+    String getResource(String resourceId);
```

Comment:
You are not retrieving resource here. Method name is misleading!

Why not to return `ExplicitAccessRecord`?

Reply:
Changed just before you commented this.